web/client/translations/data.it-IT.json

```
@@ -35,7 +35,12 @@
             "message": "Il livello non è stato ricaricato con un nome modificato"
         },
         "version": {
-            "label": "Versione"
+            "label": "Versione",
+            "message": "Messaggio",
+            "commit": "commettere",
```

not really a good way to translate this into italian, maybe we can keep english for all locales otherwise google translate suggest the verb and not the noun
```suggestion
            "commit": "Commit",
```